src/tests/ftest/io/nvme_object.yaml

```diff
@@ -0,0 +1,32 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+
+hosts:
+ test_servers:
+  - boro-A
+timeout: 2700
+server_config:
+ name: daos_server
+pool:
+ createmode:
+  mode: 511
+ createset:
+  group: daos_server
+ createsize:
+  scm_size: 1000000000
+  size:
+   - 10000000000
+   - 100000000000
+   - 800000000000
+ createsvc:
+  svcn: 1
+container:
+  object_qty: 10
```

Comment:
Is object quantity 10 is enough? We need to really fill the large number of object based on keys and pool sizes.

Reply:
That is why we need to do this in parallel. We are not going to run this test for each PR so time should not be a problem. We should push the functional test to do more things instead of small things which all ready works and DAOS is immune to this tests.

Reply:
Made the process parallel.

Reply:
I want to see how much data are we filling in the pool so getting pool info display on console helps to see how much % are we filling it up. I still think 10 object are not really enough for example record size..
1000 with 10 objects x 4096 = 40960000 , 40MB
100 with 10 objects  x 4096 = 4096000, 4 MB
50 with 10 objects x 4096 = 2048000, 2 MB
1 with 10 objects x 4096= 40960, 40K

Reply:
Shouldn't it be also multiplied with number of akeys *dkeys?
eg: 1000 recs * 10 Objs * 4096 * 10 akeys * 10 dkeys  = 4,096,000,000 = 4G

Reply:
I was under same impression as it's number of dkeys/akeys but looks like it's the size of dkey/akeys value and not the number of keys. [Code reference ](https://github.com/daos-stack/daos/blob/311ea2ef54dd8c4d2444c753c91354a2450ecfc0/src/tests/ftest/util/test_utils.py#L694 )